<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>200</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="PyDMFrame" name="RE_Connection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="PyDMFrame" name="RE_Queue_Controls">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PyDMFrame" name="RE_Worker">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="PyDMFrame" name="RE_Running">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="PyDMFrame" name="RE_Status">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>520</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>520</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>520</width>
         <height>200</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="PyDMFrame" name="RE_Plan_Execution">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMFrame</class>
   <extends>QFrame</extends>
   <header>pydm.widgets.frame</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
